packages/abi-gen-templates/partials/call.handlebars

```diff
@@ -1,3 +1,3 @@
-public {{this.tsName}} = {
+public {{this.langLocalName}} = {
```

Comment:
Thoughts on just renaming this to `name`? Now that you can specify a language when calling `abi-gen`, I think we can assume it'll spit out the corresponding `name`.

Reply:
Well, in this context, `this` is a superset of `AbiDefinition` from `@0x/ethereum-types`, and that object already has a `.name` attribute.  Furthermore, the Python template (specifically `partials.py/call.handlebars`) is currently using both: `.langLocalName` is used to give a snake-case method name to our wrapper, and `.name` is used within the wrapper to call the method on the Web3.py contract object, which exposes the method name as the original camel case.